ml-agents/mlagents/trainers/policy/tf_policy.py

```diff
@@ -255,6 +257,9 @@ def initialize_or_load(self):
             self._load_graph(self.model_path, reset_global_steps=reset_steps)
         else:
             self._initialize_graph()
+        # broadcast initial weights from worker-0
+        if broadcast_variables():
+            self.sess.run(hvd.broadcast_global_variables(0))
```

Comment:
@chriselion do you have a workaround to avoid the hvd call here? One way would be to subclass the TFPolicy, but I'm trying to avoid that.

Reply:
How about something like this?
```python
from typing import Callable

class TFPolicy(Policy):
  ...
  broadcast_global_variables: Callable[[int], None] = lambda x: None
  ...
  def initialize_or_load(self):
    ...
    TFPolicy.broadcast_global_variables(0)
```

Then when we do the distributed setup, you can set 
```python
TFPolicy.broadcast_global_variables = hvd.broadcast_global_variables
```
(so it basically acts as a "function pointer" with a no-op default)

Reply:
thanks, I'll try this.